pkg/notification/storage/v2/admin_subscription_test.go

```diff
@@ -67,19 +72,21 @@ func TestCreateAdminSubscription(t *testing.T) {
 
 			},
 			input: &domain.Subscription{
-				Subscription: &proto.Subscription{Id: "id-0"},
+				Subscription: &proto.Subscription{Id: id},
 			},
 			expectedErr: errors.New("error"),
 		},
 		{
 			desc: "Success",
 			setup: func(s *adminSubscriptionStorage) {
 				s.qe.(*mock.MockQueryExecer).EXPECT().ExecContext(
-					gomock.Any(), gomock.Any(), gomock.Any(),
+					gomock.Any(),
+					gomock.Regex("^INSERT INTO admin_subscription\\s*\\(\\s*id,\\s*created_at,\\s*updated_at,\\s*disabled,\\s*source_types,\\s*recipient,\\s*name\\s*\\)\\s+VALUES\\s+\\(\\s+(\\?,\\s*){6}\\s*\\?\\s*\\)\\s*$"),
```

Comment:
Is it bit too much to test SQL code?

Reply:
If there are columns of the same type, the positions of the arguments being passed may be swapped, leading to unexpected bugs. (e.g. `create_at` and `update_at` etc.)
It is true that the complexity of testing has increased, but it gives us peace of mind because we can test properly and accurately once.

Reply:
From the third argument, it is worth to check the correctness of the arguments. But since SQL is not dynamically generated, It would be too match, wouldn't it?